docs/guide/index.md

```diff
@@ -1,14 +1,14 @@
 # User Guide
 
 The user guide details every single feature Pydust offers. All code snippets are taken
-from our `example/` directory and are tested during CI. 
+from our `example/` directory and are tested during CI.
```

Comment:
Looks like my IDE started stripping trailing newlines..

Reply:
That's good. We shouldn't allow trailing whitespace